// SPDX-License-Identifier: GPL-3.0-or-later
// SPDX-FileCopyrightText: 2021 Jared Irwin <jrairwin@sympatico.ca>
syntax = "proto3";

package Bakaneko;

message System {
	string hostname         = 1;
	string mac_address      = 2;
	string ip_address       = 3;
	string operating_system = 4;
	string kernel           = 5;
	string architecture     = 6;
	string vm_platform      = 7;
	string icon             = 8;
};